ch04.adoc

```diff
@@ -210,304 +226,208 @@ double time(time) ;
+For observational or model-generated data that follows the Gregorian calendar and UTC (after 1972) including leap seconds, if precision to the second is important, the **`utc`** calendar is recommended in preference to the **`standard`** calendar.
+For model-generated data following the Gregorian calendar without leap seconds, the **`proleptic_gregorian`** calendar is recommended in preference to the **`standard`** calendar.
+See also <<leap-seconds>>.
 
-**`standard`**:: Mixed Gregorian/Julian calendar as defined by UDUNITS.
-This is the default.
+The possible values for the **`calendar`** attribute and the definitions of the calendars are as follows:
+
+**`standard`**:: The Gregorian--Julian calendar as defined by UDUNITS.
+This is the default calendar when there is no **`calendar`** attribute, except when the attribute `month_lengths` is present (see <<explicit-calendar>>).
 A deprecated alternative name for this calendar is **`gregorian`**.
 The Gregorian and Julian calendars have the same lengths of their months; they differ only in respect of the rules that decide which years are leap years.
 In the **`standard`** calendar, datetimes after and including 1582-10-15 0:0:0 are in the Gregorian calendar, in which a year is a leap year if either (i) it is divisible by 4 but not by 100 or (ii) it is divisible by 400.
 Datetimes before (and excluding) 1582-10-5 0:0:0 are in the Julian calendar, in which any year that is divisible by 4 is a leap year.
 Year 1 AD or CE in the **`standard`** calendar is also year 1 of the **`julian`** calendar.
 Negative years are invalid in time coordinates and reference datetimes in the **`standard`** calendar.
-In the **`standard`** calendar, 1582-10-15 0:0:0 is exactly 1 day later than 1582-10-4 0:0:0.
-Therefore datetimes in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 are invalid, and must not be used as reference in **`units`**.
++
+In the **`standard`** calendar, datetimes in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 are invalid.
+Datetimes in this range must not be used as reference in **`units`**, and the interval between the datetimes 1582-10-15 0:0:0 and 1582-10-4 0:0:0 is exactly 1 day.
 It is recommended that a reference datetime before the discontinuity should not be used for datetimes after the discontinuity, and vice-versa.
-See also <<leap-seconds>>.
++
+__The **`standard`** calendar is used both for (mostly observational) data with real-world datetimes, including leap seconds (after 1972), and for data not including leap seconds__, generated by models according to the Gregorian calendar.
+Therefore, __when using data written with the **`standard`** calendar, you should regard datetimes and time coordinates as uncertain by a number of seconds (but less than a minute),__ because leap seconds may have been counted when you expected them to have been ignored, or ignored when you expected them to have been counted.
```

Comment:
```suggestion
Therefore, when using data written with the **`standard`** calendar the datetimes and time coordinates may be uncertain by a number of seconds (but less than a minute)  because it is unknown whether leap seconds have been taken into account or not.
```